Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorTransformComponentSelection.cpp

```diff
@@ -1835,7 +1835,10 @@ namespace AzToolsFramework
 
         EditorContextMenuUpdate(m_contextMenu, mouseInteraction);
 
-        m_boxSelect.HandleMouseInteraction(mouseInteraction);
+        if (m_boxSelect.HandleMouseInteraction(mouseInteraction))
+        {
+            return true;
+        }
```

Comment:
Shoould it return when HandleMouseInteraction is true, or false? I'm just not sure what the logic behind the return is. Does the bool mean it's already handled, or that it should not be handled?

Reply:
It should return `true` here, we could update the API comment in `HandleMouseInteraction` at the interface level (or just sanity check it). This means 'input was handled, no further processing' as opposed to 'we didn't handle this, it can continue to propagate'